docs/conf/australia/2018/news/thanks-recap.rst

```diff
@@ -0,0 +1,92 @@
+packed with presentations, workshops, lightning talks and unconference sessions. Here's a quick recap of the 2 days.
+
+Conference recap
+================
+
+At over 100  people, our first conference was a great start to what should now become a regular event on the Write the Docs conference calendar. We had a mix of main stage and lightning talks, unconference sessions, workshops and a social event.
+We received a lot of positive feedback on all of these and look forward to carry them to future events. We also had some interesting feedback on some things we might improve for next time.
+
+- The main stage presentations had a good blend of topics across documentation, user experience, customer support, content strategy, coding and style guides.
+- The unconference sessions and the lightning talks was well received and prompted people to think, collaborate and talk about documentation in a more unstructured way. It also allowed people to discuss topics that were not covered fully or partially throughout the conference.
+- We received a mixed response to the workshops and this can be an area we can look at improving next year.
+Overall, we were so excited to see our first conference come together and see our awesome community gather under one roof.
+
+Conference venue
+================
+
+What better place to hold a documentation conference than a library! We had 2 rooms overlooking the sensational Port Phillip Bay right at the top floor of a public library. While the audience was not soaking in the documentation goodness, they were more than happy to stretch their legs and have a stroll through the library amidst tons of books.
+
+Conscious consumption (aka Food and Drinks)
+===========================================
+
+In the spirit of the community sharing nature of Write the Docs events, we decided to use social enterprises for our catering.
+Our food (check out the photos to see the amazing spread!) was catered for by `Streat <https://www.streat.com.au/>`__, who are tackling youth homelessness and disadvantage their way.
+When it came to beverages, we were ably supported by `Wheelly Good Coffee <https://wheellygoodcoffee.com.au/>`__, who have programs to improve the social and economic capabilities of some of Australia's most high risk young people.
+
+Videos and photos
+=================
+
+Videos of all the talks are now available on our `Youtube channel <https://www.youtube.com/playlist?list=PLy70RNJ7dYrJ1wANiqa7ObwUnoJjouQjt>`__.
+Thanks to `Darren Chan Photography <https://www.darrenchanphotography.com/>`__ (also one of our regular documentarians) for doing such a great job with these.
+You can also subscribe to our `YouTube channel <https://www.youtube.com/channel/UCPhWNEFb53x6PjnpRIYf1yg?>`_ so you don’t miss any future content!
+
+We also have photos from the event available on `Flickr <https://www.flickr.com/photos/writethedocs/albums/72157695294209550>`_, thanks to our wonderful photographer Darren Chan.
```

Comment:
Double Darren Chan? Or is this a mixup?

Reply:
No, the same person did both videos and photos

Reply:
Might worth consolidating onto one sentence then? Two thanks feels odd to me

Reply:
Updated